### DISCUSSION TITLE
Requesting for NON Profit Grant

### DISCUSSION THREAD

**Discussion by dbjpanda** [community/community#22915] [New to GitHub]
<p>Hi community members,<br>
I have a client who have NPO. I got to know about Github’s npo grant and then applied though the github’s “contact us” form sending all the required documents. But I haven’t heared anything back from Github. Can you guide me if there is any other process to apply it.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>No, there is no other process to apply for it. That team is short staffed currently because of summer vacations. They’ll get back to you as soon as they can.</p>
